src/usb_reset_interface.h

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
Can the reset interface definitions be included with a Cmake library linkage instead of a repeat inclusion of the sdk header?